.changeset/modern-planets-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-newrelic-dashboard': major
```

Comment:
The version is already `0.1.0` in the `package.json` i'd suggest that this changeset should just be `patch` here for now, so we can iterate on it and potentially make breaking changes that come out of using the plugin in anger without us getting to something like version 6.

Reply:
Okay, that makes sense. Did the change @benjdlambert